src/groups/groups.service.ts

```diff
@@ -338,9 +341,16 @@ export class GroupsService {
       // todo: create log?
       throw new GroupNameAlreadyUsedError(name);
     }
+    const avatar_owner = (await this.avatarService.findOne(parseInt(avatar)))
+      .userid;
+    if (avatar_owner == userId) {
```

Comment:
Avatar should be changed by groupOwner. However, groupOwner may be transfered to another user who did not uploaded the original avatar.

Reply:
When the new groupOwner wants to change the avatar,maybe he should upload a new avatar?

Reply:
Or you mean that we need to record history avatar?